prime-router/build.gradle.kts

```diff
@@ -30,7 +30,7 @@ import java.util.Properties
 
 plugins {
     kotlin("jvm") version "1.7.21"
-    id("org.flywaydb.flyway") version "8.5.13"
+    id("org.flywaydb.flyway") version "9.9.0"
```

Comment:
Bumps the actual version up. I have fixed the issue here that was preventing the flyway migration task from running. Specifically, in v9 Flyway moved to a different transactional model for executing scripts based on `pgBouncer` which causes any attempts to run a `create index concurrently...` to fail because concurrent index creation can't be done inside a transaction. There's a multitude of documentation on how to fix it, but it's all mostly wrong.